src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(({ sex, died }) =>
+    century
+      ? sex === 'm' && century === Math.ceil(died / 100)
+      : sex === 'm'
```

Comment:
[checklist](https://github.com/mate-academy/js_average-ages/blob/master/checklist.md) 2nd item, fix it everywhere